### ISSUE TITLE
Bytes per row options

### ISSUE THREAD

**Issue by Neoheurist** [PeeHaa/HexDump#6]
I would prefer at least an even number of bytes per row in the display or better yet a multiple of 4 bytes or better yet a multiple of 8 bytes or better yet a multiple of 16 bytes or better yet a multiple of 32 bytes

23 bytes per row is awkward / tricky for reviewing file layouts because in many cases values are aligned to a word boundary

By the way: thanks for creating this - it saved me the need to download and install a tool

**Comment by PeeHaa** [comment]
> 23 bytes per row is awkward

Yes. Makes perfect sense to setup a sane number of bytes per row. I will change this in the future. Thanks!

> thanks for creating this

You're welcome
